### PR TITLE
DOC: Separate user and dev manuals

### PR DESCRIPTION
**Pull request by pllim**
Address part of #182.

- [x] breakout into user and dev manual
- [x] add section of Jupyter widgets to dev doc; link directly to example notebooks on repo for now without rendering
- [x] move FAQ down
- [x] clean up installation instructions
- [x] remove redundant `$` prompt
- [x] shorten section titles (for ease to read on left nav bar)
- [x] misc clean-up

### CODE REVIEW

File: doc/FAQ.rst [hunk 1]
```diff
@@ -113,22 +117,22 @@ We are lucky to have several possible choices for a Python WCS package
...
 `AstLib <http://astlib.sourceforge.net/>`_,
 `Kapteyn <http://www.astro.rug.nl/software/kapteyn/>`_,
-`Starlink <https://github.com/timj/starlink-pyast>`_ and
+`Starlink <https://github.com/timj/starlink-pyast>`_, and
 :ref:`Astropy WCS <astropy:astropy-wcs>`.
 
-Kapteyn and Astropy wrap Mark Calabretta's "WCSLIB", astLib wraps
+Kapteyn and Astropy wrap Mark Calabretta's "WCSLIB", AstLib wraps
 Jessica Mink's "wcstools", and I'm not sure what Starlink uses.
-Note that Astlib and starlink require pyfits (or Astropy) to be
```
Comment (pllim) [inline doc/FAQ.rst]: PyFITS is no longer supported by STScI. That's why I removed it. FYI.

**Comment by pllim** [comment]
@ejeschke , I think I did all I meant to do with this PR. Just waiting for your addition of "how to write plugin" doc, as discussed in the 2018-09-06 tag up.

**Comment by pllim** [comment]
Is this ok to merge?

File: doc/FAQ.rst [hunk 2]
```diff
@@ -42,7 +45,7 @@ Yes.
 Can Ginga work with Gtk3?
 -------------------------
 Yes, although the performance is not on par with Gtk2 yet. Cairo for
```
Comment (ejeschke) [inline doc/FAQ.rst]: This part is out of date.  I will make a PR on your PR.  Gtk3 is the default for gtk backend now and performance is good.

File: doc/install.rst [hunk 3]
```diff
@@ -110,22 +110,21 @@ the full reference viewer.
...
 Tk
-===
+==
 
 Ginga's Tk support is limited to the viewing widget itself.  For
 overplotting (graphics) support, you will also need:
 
-* "pillow"/PIL package
-* "OpenCv" module
-* "aggdraw" module (which you can find
```
Comment (ejeschke) [inline doc/install.rst]: Note to self: `aggdraw` supports Python 3 as well now.

File: doc/install.rst [hunk 4]
```diff
@@ -139,110 +138,92 @@ HTML5 web browser
...
 Linux
 =====
 
 #. Install the necessary dependences.  If you are on a relatively recent
-   version of Ubuntu (e.g. v14.04 or later), something like the following
+   version of Ubuntu (e.g., v14.04 or later), something like the following
    will work::
 
-     $ apt-get install python-numpy python-scipy python-matplotlib \
-       python-astropy python-qt4 python-webkit python-magic git pip
```
Comment (ejeschke) [inline doc/install.rst]: Note to self: should change to qt5

File: doc/FAQ.rst (hunk 2 above)
Comment (ejeschke) [reply doc/FAQ.rst]: On second thought, @pllim, let's merge and I will make a separate small update PR to address these last changes.

**Comment by pllim** [comment]
I just realized that once this is merged, the URL for writing plugins instructions would change and the poster needs to be updated.

**Pull request merged**